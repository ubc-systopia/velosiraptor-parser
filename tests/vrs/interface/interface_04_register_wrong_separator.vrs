segment MyUnit {
  interface(base: foo) {
        reg foo [ 8 ];
        reg foo [ 8 ]
    };
}