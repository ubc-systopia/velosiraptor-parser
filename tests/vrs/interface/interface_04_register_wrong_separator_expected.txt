error: expected `}`, found `;`
     --> tests/vrs/interface/interface_04_register_wrong_separator.vrs:3:22
      |
    3 |          reg foo [ 8 ];
      |                       ^ help: add `}` here.
